StripePayments/StripePayments/Source/API Bindings/Models/Shared/LinkConsumerIncentive.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  LinkConsumerIncentive.swift
+//  StripePayments
+//
+//  Created by Till Hellmund on 10/8/24.
+//
+
+import Foundation
+
+@_spi(STP) public final class LinkConsumerIncentive: NSObject, STPAPIResponseDecodable {
+    
+    @_spi(STP) public let incentiveParams: IncentiveParams
+    @_spi(STP) public let incentiveDisplayText: String?
+    
+    @_spi(STP) public let allResponseFields: [AnyHashable : Any]
```

Comment:
Just curious, why do we want to hold on to all response fields here?

Reply:
It’s part of `STPAPIResponseDecodable`, although I’m not so sure of its usefulness here.

Reply:
Addressed in e6fee128476533b806d57b38eac4128f2e746fde.